.github/ISSUE_TEMPLATE/feedback_auto.md

```
@@ -0,0 +1,22 @@
+
+
+
+## Environment
+
+- AISH version:
+- Operating system:
+- AI Model / Provider:
+
+## Logs
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)